docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1255 @@
+        list similarly. You can likewise instantiate other components by
+        adding them to their respective lists.</para>
+      </sect3>
+
+      <sect3 id="VALYml_ECLCCServer">
+        <title>ECL Agent and ECLCC Server</title>
+
+        <para>Values of note for the ECL Agent and ECLCC Server.</para>
+
+        <para><emphasis role="bold">useChildProcess</emphasis> -- When you
+        submit a job or query to compile it gets queued and processed, with
+        this option set to true it will spawn a child process utilizing almost
+        no additional overhead in starting. Ideal for sending many small jobs
+        to compile. However, because each compile job is no longer executed as
+        an independent pod with it's own resource specifications, but instead
+        runs as a child process within the eclccserver pod itself, the
+        eclccserver pod must be defined with adequate resources for itself
+        (minimal for listening to the queue etc.) and all the jobs it may have
+        to run in parallel.</para>
+
+        <para>For example, imagine <emphasis>maxJobs</emphasis> is set to 4,
+        and 4 large queries are queued rapidly, that will mean 4 child
+        processes are launched each consuming cpu and memory within the
+        eclccserver pod. With the component configured with
+        <emphasis>useChildProcesses</emphasis> set to true, each job will run
+        in the same pod (up to the value of <emphasis>maxJobs</emphasis> in
+        parallel). Therefore with <emphasis>useChildProcesses</emphasis>
+        enabled, the component resources must be defined such that the pod has
+        enough resources to handle the resource demands of all the those jobs
+        to be able to run in parallel.</para>
+
+        <para>With useChildProcess enabled it could be rather expensive in
+        most cloud pricing models, and rather wasteful if there aren't any
+        jobs running. Instead you can set this
+        <emphasis>useChildprocess</emphasis> to false (the default) to start
+        your pod with only the required memory for the job which will be
+        disposed of when done. Now this model also has overheard, perhaps 20
+        seconds to a minute to spawn the Kubernetes cluster to process the
+        job. Which may not be ideal for an environment which is sending
+        several small jobs, but rather larger jobs which would minimize the
+        effect of the overhead in starting the Kubernetes cluster. Setting
+        <emphasis>useChildProcess</emphasis> to false better allows for the
+        possibility of dynamic scaling. For jobs which would take a long while
+        to compile, the extra (start up) overhead is minimal, and that would
+        be the ideal case to have the <emphasis>useChildProcess</emphasis> as
+        false. Setting <emphasis>useChildProcess</emphasis> to false only
+        allows 1 pod per compile.</para>
```

Comment:
Note: https://github.com/hpcc-systems/HPCC-Platform/pull/15609 changes this slightly.  There is now an option to try compiling the query within the process, but kill it if it takes more than 10 seconds, and launch a job instead.  So small jobs will run quickly, larger jobs will start a K8s job.

Reply:
Question: The change was to useChildProcess behavior, or is it adding the ChildProcessTimeLimit, and should I include some additional language for that time limit?

Reply:
The change is adding childProcessTimeLimit, and should probably have some extra language.